ms_office_macro_bleach/bleach.py

```diff
@@ -0,0 +1,76 @@
+"""This module removes any and all macros/dynamic content from MS Office files.
+
+It is part of a suite of programs developed by the AntiMalware Alliance.
+
+Visit https://github.com/Anti-Malware-Alliance for more details about our organisation and projects.
+
+Feel free to contact benjamin.mcgregor2002@gmail.com for any questions regarding this module.
+"""
+from argparse import ArgumentParser
+from os import rename, path, remove
+from zipfile import ZipFile
+from shutil import make_archive, rmtree
+
+
+# Unzip file function
+# Converts file to a .zip archive and extracts it to file directory
```

Comment:
I would add a longer comment at the beginning of the file as to why this tool is been written and a little bit on how it works, but remove the line per line comments. If the code is relatively short and clean, we dont need the line per line comment to see what is been done.

Reply:
I believe I have addressed all the other comments, but would you mind elaborating on this one?

Should I just be commenting on the functions for code cleanliness and should I remove all line code comments?

Reply:
I like your comment at the beginning. I would remove the online comments. If you have a large function, you might want to add a comment to the beginning of the function. Its one of the recommendation from Robert Martin, Author of Clean Code.